src/types/local-transcript-manager.ts

```diff
@@ -1,17 +1,39 @@
 // Manage the transcript database locally
-import {StudentID, Student, Course, Grade, CourseGrade, Transcript, TranscriptManager} from './transcript';
+import {
+  StudentID,
+  Student,
+  Course,
+  Grade,
+  CourseGrade,
+  Transcript,
+  TranscriptManager,
+  CourseDescription,
+} from "./transcript";
 // manage the transcript database
 
 // the database of transcript
 let allTranscripts: Transcript[] = [];
+let allCoursesDescriptions: CourseDescription[] = [];
```

Comment:
no need for initialization here if you are initializing it at line 20,
Also, this could probably be a dictionary, instead of a list, as it has better performance than a list